src/components/chip-set/chip-set.tsx

```diff
@@ -177,6 +177,16 @@ export class ChipSet {
     @Prop({ reflect: true })
     public delimiter: string = null;
 
+    /**
+     * For chip-set of type `input`, defines whether the input field should have autocomplete enabled.
+     * Read more about the `autocomplete` attribute
+     * [here](https://developer.mozilla.org/en-US/docs/Web/HTML/Attributes/autocomplete).
+     *
+     * If no value is set, a random value is generated to avoid browser suggestions.
+     */
+    @Prop({ reflect: true })
+    public autocomplete: string;
+
     /**
```

Comment:
In the Chip set and Picker (which is a consumer of `limel-chip-set`) the default `autocomplete` suggestions of some browsers like Microsoft Edge, create user experience issues.

The experience is more annoying in the Picker, as it has its own list of suggestions. There is no point in overlaying another list on top of this list. 

By generating a random `autocomplete` value and defaulting to that in the Chip set component, we prevent browsers from offering autocomplete suggestions, while still using "off" to indicate that autocomplete is not desired. 

While there might be scenarios that the Chip set can benefit from auto suggestions. Therefore, I kept the `autocomplete` as a `prop` here. But I am not sure if this is the right way. Maybe we should just not use any prop at all, and just directly default to ``off ${crypto.randomUUID()}`;`

Reply:
Yeah, I think we should avoid forcing autocomplete to be disabled without at least leaving the consumer the option to enable it again.

> Maybe we should just not use any prop at all, and just directly default to `off ${crypto.randomUUID()};`

Just to clarify some terminology here:
- If we would have no prop at all, that would be "hardcoding" the value.
- If we use `off ${crypto.randomUUID()}` as the default value, it would look like this:
  ```ts
  @Prop({ reflect: true })
  public autocomplete?: string = `off ${crypto.randomUUID()}`;
  ```

We should use the latter 🙂

Reply:
OK. I change how I default the value. My thought was we don't have to explain to the consumers why we use the random value. That's why I took it further down in the code. Anyhow, this one is also good. Not sure we need to explain why we default to what we default to. But it also doesn't hurt.